flight_phase_files/HALO_RF03_20200124_info.yaml

```
@@ -19,59 +19,115 @@ remarks:
 - Circle pattern, in the end excursion towards NTAS.
 - Deeper convection in the SE circle, spreading cold pools and stratiform clouds.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-24 09:50:17
-  end: 2020-01-24 10:48:51
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-24 10:48:51
-  end: 2020-01-24 11:00:08
-- kind: circle
-  name: c2
-  start: 2020-01-24 11:00:08
-  end: 2020-01-24 12:00:00
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-24 12:00:00
-  end: 2020-01-24 12:15:10
-- kind: circle
-  name: c3
-  start: 2020-01-24 12:15:10
-  end: 2020-01-24 13:10:03
-- kind: circle_break
-  name: cb3to4
-  start: 2020-01-24 13:10:03
-  end: 2020-01-24 13:30:05
-- kind: circle
-  name: c4
-  start: 2020-01-24 13:30:05
-  end: 2020-01-24 14:25:07
-- kind: circle_break
-  name: cb4to5
-  start: 2020-01-24 14:25:07
-  end: 2020-01-24 14:40:08
-- kind: circle
-  name: c5
-  start: 2020-01-24 14:40:08
-  end: 2020-01-24 15:35:05
-- kind: circle_break
-  name: cb5to6
-  start: 2020-01-24 15:35:05
-  end: 2020-01-24 15:50:06
-- kind: circle
-  name: c6
-  start: 2020-01-24 15:50:06
-  end: 2020-01-24 16:45:10
-- kind: straight_leg
-  name: leg1_outbound_NTAS
+- kinds:
+  - circle
+  name: Circle-1
+  irregularities: null
+  segment_id: HALO-0124c1
+  start: 2020-01-24 09:49:17
+  end: 2020-01-24 10:46:35
+  good_dropsondes: null
```

If I am not mistaken there are 11 sondes in that circle minus 2 at 09:52:10 and 10:06:12 with no launch detect according to your no-launch-detect file. However, those 2 times do not appear in the list generated by reports.py. So is the number of good sondes 11?
```suggestion
  good_dropsondes: 11
```

---

Somehow, my good_dropsondes list didn't make it into the loop to add attributes to the ResearchFlightSegment. I have changed the notebook now, and the good_dropsondes issue should be solved now.

---

The 11 sondes is because one of the bad sondes (09:52) is missing from the unified dataset already. Plus the other bad sonde (10:06) is present. So, I would say the good sondes number would be 10, in this case.